ArcGISRuntimeSDKQt_QMLSamples/EditData/EditWithBranchVersioning/EditWithBranchVersioning.qml

```diff
@@ -114,11 +114,11 @@ Rectangle {
 
     ServiceGeodatabase {
         id: sgdb
-        url: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/DamageAssessment/FeatureServer"
+        url: "https://sampleserver7.arcgisonline.com/server/rest/services/DamageAssessment/FeatureServer"
 
         credential: Credential {
             username: "editor01"
-            password: "editor01.password"
+            password: "S7#i2LWmYH75"
```

Comment:
@jared-2016 @ldanzinger is
```
credential: Credential {
  ...
}
```
 preferred over:
```
Credential {
  ...
}
```
?

Reply:
Both are perfectly acceptable and work the same. The first is very explicit and the second is more concise and makes use of default properties. I tend to use the default properties when available